etc/ProdOfflineConfiguration.py

```diff
@@ -234,7 +236,7 @@
                  data_tiers=["FEVT"],
                  write_dqm=True,
                  alca_producers=["SiStripPCLHistos", "SiStripCalZeroBias", "TkAlCosmics0T",
-                                 "SiPixelCalZeroBias",
+                                 "SiPixelCalZeroBias", "PromptCalibProdSiPixelLAMCS",
```

Comment:
@germanfgv this should be:
```suggestion
                 alca_producers=["SiStripPCLHistos", "SiStripCalZeroBias", "TkAlCosmics0T",
                                 "SiPixelCalZeroBias", "SiPixelCalCosmics",
                                 "PromptCalibProdSiStrip", "PromptCalibProdSiPixelLAMCS"
                                ],
```